react/tic-tac-toe/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "tic-tac-toe",
+  "version": "1.0.0",
+  "private": true,
+  "dependencies": {
+    "@redux-beacon/google-analytics": "^1.1.1",
+    "apisauce": "^1.0.0",
+    "babel-plugin-module-resolver": "^3.1.1",
+    "connected-react-router": "^5.0.1",
+    "history": "^4.7.2",
+    "i18next": "^12.1.0",
+    "lodash": "^4.17.11",
+    "node-sass": "^4.10.0",
+    "postcss": "^7.0.6",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-redux": "^5.1.1",
+    "react-router": "^4.3.1",
+    "react-router-dom": "^4.3.1",
+    "react-router-redux": "^4.0.8",
+    "react-scripts": "2.1.1",
+    "react-spinkit": "^3.0.0",
+    "redux": "^4.0.1",
+    "redux-beacon": "^2.0.3",
+    "redux-form": "^7.4.2",
+    "redux-recompose": "^2.1.0",
+    "redux-thunk": "^2.3.0",
+    "wolox-equalizer": "0.0.3"
+  },
+  "scripts": {
+    "start": "react-app-rewired start ",
+    "build": "react-app-rewired build ",
+    "test": "react-app-rewired test --env=jsdom",
+    "eject": "./node_modules/react-scripts/bin/react-scripts.js eject",
+    "deploy": "react-app-rewired build ",
+    "lint": "./node_modules/eslint/bin/eslint.js src",
+    "lint-fix": "./node_modules/eslint/bin/eslint.js src --fix",
+    "lint-diff": "git diff --name-only --cached --relative | grep \\.js$ | xargs eslint",
```

Comment:
I'd add `--diff-filter=ACM` for `git diff` so husky doesn't crash while running eslint on deleted files